Library/Formula/connect.rb

```diff
@@ -0,0 +1,18 @@
+class Connect < Formula
+  desc "provides SOCKS and HTTPS proxy support to SSH."
+  homepage "https://bitbucket.org/gotoh/connect/wiki/Home"
+  url "https://bitbucket.org/gotoh/connect/get/1.104.tar.gz"
+  sha256 "6ca5291ea8e4a402d875a5507ac78d635373584fd1912f2838b93e03b99730c8"
+
+  def install
+    # remove check for windows version that throws error
+    inreplace "Makefile", "WINVER := $(shell ver)", ""
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!